commit ec6a82cf9f8869f0c4e42842f36e0f5c6517c8aa
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Tue Apr 12 16:12:12 2022 +0200

    HACK: unconditionally compile in gcc_personality_v0
    
    Looks like HAVE_UNWIND_H is not accounted for when doing the
    runtimes build, but this is the only place where it's checked.
    
    Unconditionally compile it in because some things use it.

diff --git a/compiler-rt/lib/builtins/CMakeLists.txt b/compiler-rt/lib/builtins/CMakeLists.txt
index 14673ceae..b1f27ee95 100644
--- a/compiler-rt/lib/builtins/CMakeLists.txt
+++ b/compiler-rt/lib/builtins/CMakeLists.txt
@@ -215,12 +215,10 @@ if(APPLE)
   )
 endif()
 
-if (HAVE_UNWIND_H)
-  set(GENERIC_SOURCES
-    ${GENERIC_SOURCES}
-    gcc_personality_v0.c
-  )
-endif ()
+set(GENERIC_SOURCES
+  ${GENERIC_SOURCES}
+  gcc_personality_v0.c
+)
 
 if (NOT FUCHSIA)
   set(GENERIC_SOURCES
diff --git a/compiler-rt/lib/builtins/gcc_personality_v0.c b/compiler-rt/lib/builtins/gcc_personality_v0.c
index afb9e2e11..136460448 100644
--- a/compiler-rt/lib/builtins/gcc_personality_v0.c
+++ b/compiler-rt/lib/builtins/gcc_personality_v0.c
@@ -9,7 +9,7 @@
 #include "int_lib.h"
 #include <stddef.h>
 
-#include <unwind.h>
+#include "../../../libunwind/include/unwind.h"
 #if defined(__arm__) && !defined(__ARM_DWARF_EH__) &&                          \
     !defined(__USING_SJLJ_EXCEPTIONS__)
 // When building with older compilers (e.g. clang <3.9), it is possible that we
